core/src/main/java/com/google/instrumentation/trace/SpanBuilder.java

```diff
@@ -102,18 +102,17 @@
   private final SpanFactory spanFactory;
   private final String name;
   private final StartSpanOptions startSpanOption = new StartSpanOptions();
+  private Span parentSpan;
   private SpanContext parentSpanContext;
```

Comment:
personally I like remote parent being a part of the context, as that allows you to separate the concern cleanly (context being demarshalled from a remote source is pretty easy to identify)